app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -56,6 +56,19 @@ echo
   "connectefi #{connectefi_option}" if connectefi_option
 %>
 
+if [ "${lockdown}" == "y" ]; then
+
+  if [ "${default}" == "local" ]; then
```

Comment:
I added another condition to not hard override the default entry in the case that `default_pxe_item_local` and/or `default_pxe_item_global` was set by the user in the settings.